build.gradle

```diff
@@ -23,6 +23,7 @@ apply from: "${rootProject.projectDir}/publish-maven.gradle"
 group = 'org.springframework.integration'
 
 repositories {
+	mavenLocal()
```

Comment:
Right, because I tested it against the build in my just merged PR in SI https://github.com/spring-projects/spring-integration/pull/2455.

Now this change can be reverted.